firmware/common2015/utils/logger/logger.hpp

```diff
@@ -24,6 +25,8 @@
 #define LOG(lvl, ...) \
     log(lvl, __BASE_FILE_NAME__, __LINE__, __func__, __VA_ARGS__)
 
+#define S_LOG(lvl) LogHelper(lvl, __BASE_FILE_NAME__, __LINE__, __func__)
```

Comment:
One last thing.  Can you add an example usage above this macro/function?  Some people may not be very familiar to stream-based logging or how this is setup, so it'd probably be helpful to show quick example.